src/main/kotlin/net/atos/zac/signalering/SignaleringenMailHelper.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Atos, 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+package net.atos.zac.signalering
+
+import jakarta.inject.Inject
+import net.atos.zac.identity.IdentityService
+import net.atos.zac.mail.model.MailAdres
+import net.atos.zac.mailtemplates.MailTemplateService
+import net.atos.zac.mailtemplates.model.Mail
+import net.atos.zac.mailtemplates.model.MailTemplate
+import net.atos.zac.signalering.model.Signalering
+import net.atos.zac.signalering.model.SignaleringDetail
+import net.atos.zac.signalering.model.SignaleringTarget
+import net.atos.zac.signalering.model.SignaleringType
+
+class SignaleringenMailHelper @Inject constructor(
+    private val identityService: IdentityService,
+    private val mailTemplateService: MailTemplateService,
+) {
+
+    fun getTargetMail(signalering: Signalering): SignaleringTarget.Mail? =
+        when (signalering.targettype!!) {
```

Comment:
Probably the targettype field should not be nullable. Because if it is this would cause a nullpointer anyway.

`!!` is almost never needed in Kotlin except in test code sometimes. :-)

Reply:
Replaced it with `else -> null` which seems better fit